scylla-gdb.py

```diff
@@ -1383,6 +1383,23 @@ def invoke(self, arg, from_tty):
             gdb.write('{:5} (replica::database*){}\n'.format(shard, db))
 
 
+class scylla_commitlog(gdb.Command):
```

Comment:
Some documentation please. Even a single sentence helps a lot, because it shows up in the `help scylla` output.